pkg/config/config.go

```diff
@@ -44,11 +44,24 @@ func (d *Duration) UnmarshalJSON(b []byte) error {
 		}
 		d.Duration = dur
 		return nil
+	case map[string]interface{}:
+		//NOTE
+		//not sure if this is gonna be an issue
+		//idk how to properly check if its string or float
+		d.Duration = time.Duration(value["Duration"].(float64))
+		return nil
```

Comment:
```suggestion
```
to solve this just add a custom Marshler for the duration
```
func (d Duration) MarshalJSON() ([]byte, error) {
	return []byte("\"" + d.String() + "\""), nil
}
```
I tested it locally and it works.